app/templates/public/index.hbs

```diff
@@ -12,4 +12,8 @@
       {{public/speaker-list speakers=nonFeaturedSpeakers}}
     </div>
   </div>
+  <h1>Getting Here</h1>
+  <div class="event-location">
```

Comment:
just keep the class as `location` since this is already inside a div called as `event`